app/org/sagebionetworks/bridge/dynamodb/OptionLookup.java

```diff
@@ -35,5 +34,11 @@ public String get(String healthCode) {
     public SharingScope getSharingScope(String healthCode) {
         return SharingScope.valueOf(get(healthCode));
     }
+    
+    public Set<String> getDataGroups(String healthCode) {
+        String value = map.get(healthCode);
+        Set<String> set = BridgeUtils.commaListToSet(value);
+        return (set.isEmpty()) ? null : set;
+    }
```

Comment:
Do we really want to convert empty sets into nulls? Everywhere else, we use an empty set.